modules/check_in/app/services/travel_claim/client.rb

```diff
@@ -58,6 +58,7 @@ def token
     #
     def submit_claim(token:, patient_icn:, appointment_date:)
       connection(server_url: claims_url).post("/#{claims_base_path}/api/ClaimIngest/submitclaim") do |req|
+        req.options.timeout = 120 if Flipper.enabled?(:check_in_experience_travel_claim_increase_timeout)
```

Comment:
Do we ever see this take up to 120 seconds? Wondering if we start with something lower instead? Can we get some stats on how often it takes > 60 sec please

Reply:
> Do we ever see this take up to 120 seconds? Wondering if we start with something lower instead? Can we get some stats on how often it takes > 60 sec please
Over last month or so that we have been piloting with 1 clinic, we have seen the timeout happen 3-4 times (out of a total of about 350 calls). So not huge, but we'd still like to handle this.

> Just throwing this out there too... would this benefit from being moved to a job instead? That way, we would get automated retries and additionally, moving it to a job wouldn't hog the thread for extended periods of time

This client is actually used by a [background worker](https://github.com/department-of-veterans-affairs/vets-api/blob/ca023296674512f8cf0a18502e28da1e48ff411a/modules/check_in/app/workers/check_in/travel_claim_submission_worker.rb#L39). The challenge is that we cannot resubmit because then the claim would be considered duplicate, and the BTSSS API doesn't provide a way (yet) to check the status of a claim that we have previously filed.